thirdparty/openapi3/schemas/Credential.yaml

```diff
@@ -0,0 +1,22 @@
+title: Credential
+type: object
+description: |
+  A credential used to allow a user to prove their identity and
+  access to an account with a DFSP.
+properties:
+  id:
+    type: string
```

Comment:
question for auth team: Is this supposed to be a `CorrelationId`

Reply:
Nah I don't think so - this is the `KeyHandleId` which comes back from the FIDO API on the device. I'm happy to keep it as a string.

Reply:
Thanks for clarifying.